index.html

```diff
@@ -113,13 +113,435 @@ <h3 class="hero__form-title">Book Appointment</h3>
+                                <li> Fastest way to organize</li>
+                                <li> Work at the speed of thought.</li>
+                                <a href="#">Learn More</a>
+                            </ul>
+                        </div>
+                    </div>
+                    <!-- card end  -->
+                    <!-- card start  -->
+                    <div class="leading__cards-card">
+                        <div class="leading__cards-card-header">
+                            <div class="card-icon">
+                                <img class="person-icon" src="./assets/icons/people.svg" alt="person-icon">
+                                <!-- mobile only icon -->
+                                <img class="frame-icon" src="/assets/icons/frame-icon.svg" alt="frame-icon">
+                            </div>
+                            <h5 class="card-title">
+                                Emergency Case
+                            </h5>
+                        </div>
+                        <div class="leading__cards-card-info">
+                            <ul>
+                                <li> The best products start
+                                    with Figma</li>
+                                <li> Design with real data</li>
+                                <li> Lightning fast prototyping</li>
+                                <li> Fastest way to organize</li>
+                                <li> Work at the speed of thought.</li>
+                                <a href="#">Learn More</a>
+                            </ul>
+                        </div>
+                    </div>
+                    <!-- card end  -->
+                    <!-- card start  -->
+                    <div class="leading__cards-card">
+                        <div class="leading__cards-card-header">
+                            <div class="card-icon">
+                                <img class="person-icon" src="./assets/icons/people.svg" alt="person-icon">
+                                <!-- mobile only icon -->
+                                <img class="frame-icon" src="/assets/icons/frame-icon.svg" alt="frame-icon">
+                            </div>
+                            <h5 class="card-title">
+                                Emergency Case
+                            </h5>
+                        </div>
+                        <div class="leading__cards-card-info">
+                            <ul>
+                                <li> The best products start
+                                    with Figma</li>
+                                <li> Design with real data</li>
+                                <li> Lightning fast prototyping</li>
+                                <li> Fastest way to organize</li>
+                                <li> Work at the speed of thought.</li>
+                                <a href="#">Learn More</a>
```

Comment:
you probably want the anchore element to be outside the ul since it is not a list item, or wrap it inside li if it belongs to the list